docs/IntegrationWithExistingApps.md

```diff
@@ -248,6 +248,30 @@ end
 
 <block class="objc swift" />
 
+> **Note:** From React Native `0.42.0` you have to explicitly include Yoga in your Podfile. Here is an example:
+
+```
+source 'https://github.com/CocoaPods/Specs.git'
+
+# Required for Swift apps
+platform :ios, '8.0'
+use_frameworks!
+
+# The target name is most likely the name of your project.
+target 'your-project' do
+
+  react_native_path = "../node_modules/react-native"
+  pod 'React', :path => react_native_path, :subspecs => [
+    'Core',
+    'RCTText',
+    'RCTNetwork',
+    'RCTWebSocket',
+  ]
+  # Explicitly include Yoga
```

Comment:
The existing Podfile are extracted from projects `NumberTileGame ` and `swift-2048` which are not using react-native `0.42.0` so I didn't want to add confusion. That's why I added a new Podfile example block and added a Note to explain why.